frontend/src/__tests__/cypress/cypress/tests/mocked/pipelines/pipelines.cy.ts

```diff
@@ -54,100 +54,6 @@ describe('Pipelines', () => {
     pipelinesGlobal.findConfigurePipelineServerButton().should('be.enabled');
   });
 
-  it('Configure pipeline server when data connection exist', () => {
-    initIntercepts({ isEmpty: true });
-
-    cy.interceptK8s(
-      'POST',
-      {
-        model: SecretModel,
-        ns: projectName,
-      },
-      mockSecretK8sResource({ namespace: projectName }),
-    ).as('createSecret');
-
-    cy.interceptK8s(
-      'POST',
-      DataSciencePipelineApplicationModel,
-      mockDataSciencePipelineApplicationK8sResource({
-        namespace: projectName,
-      }),
-    ).as('createDSPA');
-
-    pipelinesGlobal.visit(projectName);
-    cy.interceptK8sList(
-      SecretModel,
-      mockK8sResourceList([mockSecretK8sResource({ namespace: projectName })]),
-    ).as('refreshSecrets');
-
-    cy.interceptK8sList(
-      {
-        model: SecretModel,
-        ns: projectName,
-      },
-      mockK8sResourceList([
-        mockSecretK8sResource({ s3Bucket: 'c2RzZA==', namespace: projectName }),
-      ]),
-    );
-    pipelinesGlobal.findConfigurePipelineServerButton().should('be.enabled');
-    pipelinesGlobal.findConfigurePipelineServerButton().click();
-    configurePipelineServerModal.selectDataConnection('Test Secret •••••••••••••••••');
```

Comment:
When you bring back this test -- I am curious if you thought to go through and rename some of these utilities `selectDataConnection` could get a rename too `selectViableConnection` or something like that.